test/e2e/common/runtime.go

```diff
@@ -328,39 +333,50 @@ while true; do sleep 1; done
 
 			It("should not be able to pull image from invalid registry [NodeConformance]", func() {
 				image := "invalid.com/invalid/alpine:3.1"
-				imagePullTest(image, false, v1.PodPending, true)
+				imagePullTest(image, false, v1.PodPending, true, false)
 			})
 
 			It("should not be able to pull non-existing image from gcr.io [NodeConformance]", func() {
 				image := "k8s.gcr.io/invalid-image:invalid-tag"
-				imagePullTest(image, false, v1.PodPending, true)
+				imagePullTest(image, false, v1.PodPending, true, false)
 			})
 
-			// TODO(claudiub): Add a Windows equivalent test.
 			It("should be able to pull image from gcr.io [LinuxOnly] [NodeConformance]", func() {
 				image := "gcr.io/google-containers/debian-base:0.4.1"
-				imagePullTest(image, false, v1.PodRunning, false)
+				imagePullTest(image, false, v1.PodRunning, false, false)
+			})
+
+			It("should be able to pull image from gcr.io [NodeConformance]", func() {
+				framework.SkipUnlessNodeOSDistroIs("windows")
+				image := "gcr.io/kubernetes-e2e-test-images/windows-nanoserver:v1"
+				imagePullTest(image, false, v1.PodRunning, false, true)
 			})
 
 			It("should be able to pull image from docker hub [LinuxOnly] [NodeConformance]", func() {
 				image := "alpine:3.7"
-				imagePullTest(image, false, v1.PodRunning, false)
+				imagePullTest(image, false, v1.PodRunning, false, false)
 			})
 
-			It("should be able to pull image from docker hub [WindowsOnly] [NodeConformance]", func() {
+			It("should be able to pull image from docker hub [NodeConformance]", func() {
 				framework.SkipUnlessNodeOSDistroIs("windows")
 				image := "e2eteam/busybox:1.29"
-				imagePullTest(image, false, v1.PodRunning, false)
+				imagePullTest(image, false, v1.PodRunning, false, true)
```

Comment:
FWIW I get why this is changed to true, but busybox/windows test?  This specific test is a little weird and needs better description IMO.  I don't understand why this is windows specific, or why the logic exists above.

Reply:
the busybox here that is pulled will end up being Windows based (manifest list), though a comment is not present about this in the code.

@bclau might as well use an official Microsoft image here?

Reply:
why not push the same `v1` image we now have on GCR to the docker hub account....?

Reply:
Indeed, the pulled image is manifest list with only Windows images. This is the Windows equivalent test of "should be able to pull image from docker hub [LinuxOnly] [NodeConformance]"

Reply:
@PatrickLang let's wait for this conversation to be resolved before we wrap up this PR

Reply:
@neolit123  I understand that using the official msft image is better and we can moving forward but we are using busybox in many tests so far. we don't want to make all these changes now 2 days before code freeze. are you ok with that? we can file an issue and fix for 1.15

Reply:
@michmike i don't want to block anything.
@timothysc questioned the usage of busybox, so i proposed solutions.

also i'm not an approver for `/test` so the final call is not mine.

Reply:
i agree with your proposal, i was just asking to postpone it a little bit :)